src/pages/VenuePage/VenuePage.tsx

```diff
@@ -70,12 +70,12 @@ const checkSupportsPaidEvents = (template: VenueTemplate) =>
   template === VenueTemplate.jazzbar;
 
 export const VenuePage: React.FC = () => {
-  const venueId = useVenueId();
-  const venue = useSelector(currentVenueSelector);
-  const analytics = useAnalytics({ venue });
-  const { world, isLoaded: isWorldLoaded } = useCurrentWorld({
-    worldId: venue?.worldId,
-  });
+  const { worldSlug } = useWorldParams();
+  const spaceSlug = useSpaceParams();
+  const { space, isLoaded } = useSpaceBySlug(spaceSlug);
+  const venueId = useMemo(() => space?.id, [space?.id]);
```

Comment:
Same comment as !!!venueId!!! but also, why is this memoized?

Reply:
I was testing if this introduces extra re-rendering. It was for dev/testing purposes but I forgot about it.